src/test/java/org/jabref/logic/integrity/UnicodeNormalFormCCheckTest.java

```diff
@@ -0,0 +1,34 @@
+package org.jabref.logic.integrity;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class UnicodeNormalFormCCheckTest {
+    UnicodeNormalFormCCheck checker = new UnicodeNormalFormCCheck();
+    BibEntry entry = new BibEntry();
+
+    @Test
+    void checkWithNormalizedStringShouldReturnEmptyList() {
+        entry.setField(StandardField.TITLE, "Some Title");
+        entry.setField(StandardField.AUTHOR, "John Doe");
+
+        assertEquals(Collections.emptyList(), checker.check(entry));
+    }
+
+    @Test
+    void checkWithNonNormalizedStringShouldReturnIntegrityMessage() {
```

Comment:
I have no clue on the normal forms. The [JavaDoc of the normalizer](https://docs.oracle.com/javase/6/docs/api/java/text/Normalizer.html#isNormalized(java.lang.CharSequence,%20java.text.Normalizer.Form)) gives other examples:

> Characters with accents or other adornments can be encoded in several different ways in Unicode. For example, take the character A-acute. In Unicode, this can be encoded as a single character (the "composed" form):
>
>      U+00C1    LATIN CAPITAL LETTER A WITH ACUTE
>
> or as two separate characters (the "decomposed" form):
>
>      U+0041    LATIN CAPITAL LETTER A
>      U+0301    COMBINING ACUTE ACCENT

Can you add a test for that, too? I assume that the first one is OK and the last one is NOT. Please use `@ParameterizedTest` and a `@MethodSource` to provide the input. Just pass the "title" as string to be set as title. The field "Author" can stay as is.